client/jmx-adapter/pom.xml

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2009-2020 Roland Huss
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>jmx-adapter</artifactId>
+    <version>1.6.3-SNAPSHOT</version>
+    <name>jolokia-jmx-adapter</name>
+    <packaging>bundle</packaging>
```

Comment:
Question: Not really sure about the OSGI stuff. Though supporting that format may be beneficial when using from a modular system such as jvisualvm (Netbeans RCP) and jmc (Eclipse RCP) but get warnings that neither jolokia-core, http client nor commons logging are OSGI compliant. Can change to just building thin jar and fat jar if that is an issue